### PR TITLE
ALT-194 Add references to WLM recovery procedures

### PR DESCRIPTION
**Pull request by dgloe-hpe**
# Description

Add references to the WLM restore procedures from the CPE installation guide.

# Checklist

- [x] If I added any command snippets, the steps they belong to follow the prompt conventions (see [example][1]).
- [x] If I added a new directory, I also updated `.github/CODEOWNERS` with the corresponding team in [Cray-HPE][2].
- [x] My commits or Pull-Request Title contain my JIRA information, or I do not have a JIRA.

 
[1]: https://github.com/Cray-HPE/docs-csm/blob/main/introduction/documentation_conventions.md#using-prompts
[2]: https://github.com/Cray-HPE/teams

### CODE REVIEW

File: operations/System_Recovery.md [hunk 1]
```diff
@@ -574,4 +576,57 @@ The latest CSM documentation has been installed on the master nodes. See [Check
...
+*10.3.11 Restore Slurm Accounting Database from Backup* and
+*10.3.13 Restore Slurm Spool Directory from Backup* in the
+**HPE Cray Programming Environment Installation Guide: CSM on HPE Cray EX Systems (S-8003)**.
+
+After restoring Slurm data from backup, check that the procedure was successful.
+
+1. (`uan#`) Check the accounting records.
+
+   ```bash
+   sacct -S <date>
```
Comment (kimjensen-hpe) [inline operations/System_Recovery.md:591]: Would `sacct -a` be a simpler check?  How would one determine the <date> to use if checking foraccount starttimes? 
Also is seems that the <date> is based on CDT+12??  Maybe for those the use slurm this is all a known - so disregard if you feel customers will know how to select a valid <date> for this check.
```
uan01:~ # date
Fri 31 Mar 2023 05:52:54 PM CDT
uan01:~ # sacct -S 2023-03-31T17:55:00
sacct: error: Start time (2023-03-31T17:55:00) requested is after end time (2023-03-31T17:53:15).
uan01:~ # date
Fri 31 Mar 2023 05:53:17 PM CDT
uan01:~ # sacct -S 2023-03-31T17:53:00
JobID           JobName  Partition    Account  AllocCPUS      State ExitCode
------------ ---------- ---------- ---------- ---------- ---------- --------
692208         msgget01      workq                    32    RUNNING      0:0
```

File: operations/System_Recovery.md [hunk 2]
```diff
@@ -574,4 +576,57 @@ The latest CSM documentation has been installed on the master nodes. See [Check
...
+   ```
+
+1. (`uan#`) Check queued jobs.
+
+    ```bash
+   squeue
```
Comment (kimjensen-hpe) [inline operations/System_Recovery.md:597]: If the backup was not successful, would `squeue` be empty?  Is there an example that can be added to show what might be seen if the restore was not successful?  Same question for `sinfo `below` as well as the pbs commands.

Comment (dgloe-hpe) [reply operations/System_Recovery.md:597]: The queue should match the state when it was backed up. So if the queue was empty, it should still be empty. The only way you could tell that a restore failed would be if there were jobs in the queue when it was backed up, but they don't show up after the backup/restore.

File: operations/System_Recovery.md (hunk 1 above)
Comment (dgloe-hpe) [reply operations/System_Recovery.md:591]: `sacct -a` will only show the jobs run during the current day. If the backup was done more than a day before the restore, it won't show any jobs. To make sure the backup/restore worked, you want to show jobs from before the backup was made.

Comment (kimjensen-hpe) [reply operations/System_Recovery.md:591]: would it make sense to then say that the <date> should be from before the backup was taken?  And what TZ is that date based on?

**Review by kimjensen-hpe — APPROVED** [review]

**Pull request merged**